### ISSUE TITLE
Wheel doesn't save size or position after change it

### ISSUE THREAD

**Issue by gustavoalara** [pcca-matrix/PCCA-Layout#60]
Hi,

I've trying to change the position and size of my hyperspin themes wheels using PCCA, but after change the position, size and number of elements of the wheel and back to the theme, the wheel doesn't change anything. I tried it on several themes with the same result

Could be something that I've doing wrong?

**Comment by pcca-matrix** [comment]
Hi,
First , take into consideration that the wheel settings is not global. 
ex: if you change the wheel settings in the main menu , it will not be changed on systems themes.
 
Can you check if the changes are saved in the ini file ? you'll find it in the directory  "layouts/pcca/settings/system_name.ini"
or try to delete the ini, it will be recreated at next restart.

Let me know if it works.

**Comment by gustavoalara** [comment]
Hi,

Yeah, I know the wheel settings is per system/display.

I check the ini file, regarding wheel, PCCA saves the position offset but not the H or V size I configure on the theme menu. (I've checked deleting first the ini file to force PCCA create a new one).

So, it looks like your nut doesn't save the wheel settings on the ini file. I'll very thankful if you can check it (I can change the sizes modifying your nut, but I prefer not to do that in order to maintain your code as-is)

**Comment by pcca-matrix** [comment]
I don't understand what you mean by wheel size? 
what is currently adjustable in the edit menu is the number of wheel slots, the x,y pos and the rotation of the entire wheel surface but not the "size".

maybe you talk about the size of the wheel logos?

if so this option is currently not available but it would be a good idea to add it to the wheel edit menu.

**Comment by gustavoalara** [comment]
Exactly, I mean the wheel logos, sorry for my bad explanation. But is curious, because you can change the size of the wheel logos using the numpad keys in the same way you can do on the special artwork submenus...

**Comment by pcca-matrix** [comment]
yes it could be done this way but since it resizes the complete surface the wheel logos may end up outside the display area ! 
anyway i had planned to rewrite the whole wheel part , i would do that at the same time

**Comment by pcca-matrix** [comment]
the new wheel module allow saving the wheel size !
